packages/design-tokens/docs/StoryBoard.stories.tsx

```diff
@@ -251,20 +262,19 @@ const StoriesContainer = (props: {
       {shouldRender && (
         <Stack key={key}>
           {Object.entries(props.storyModule).map(([k, V]) => {
-            // Ignore exports that are Story Metadata (which we can detect by the presence of a "title" prop)
-            if ("title" in V) return null
-            const parameters = "story" in V ? V.story.parameters : V.parameters
+            if (isStoryMeta(V)) {
```

Comment:
In my PR for the Vulnerability Checking (KDS-99) I ran into some problems here as well, whoever merges in 2nd will likely have to make some changes again here ;)

Reply:
Thanks, It's KDS-104 if you want to see the ticket.